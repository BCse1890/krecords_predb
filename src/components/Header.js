import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Student Record Webpage</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>Homepage</NavLink>
        <NavLink to="/add" activeClassName="is-active">Create Record</NavLink>
        <NavLink to="/edit" activeClassName="is-active">Edit Record</NavLink>
        <NavLink to="/help" activeClassName="is-active">Help</NavLink>
        <NavLink to="/contact" activeClassName="is-active">Registration details</NavLink> 
    </header>
);

export default Header;