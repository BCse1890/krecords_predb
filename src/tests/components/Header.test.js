import React from 'react';
import{ shallow } from 'enzyme';
import '../setUpTests';
import Header from '../../components/Header';

test('render Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
})