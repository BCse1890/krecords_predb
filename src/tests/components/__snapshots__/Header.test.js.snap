// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Header 1`] = `
<header>
  <h1>
    Student Record Webpage
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    Homepage
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/add"
  >
    Create Record
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/edit"
  >
    Edit Record
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
    Help
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/contact"
  >
    Registration details
  </NavLink>
</header>
`;
